source/includes/steps-generated-models-cli.yaml

```
@@ -0,0 +1,46 @@
+title: Pull the Latest Version of Your App
+ref: check-for-the-latest-version-of-your-app
+content: |
+  To ensure the CLI is accessing the latest version of your app, run the following:
+
+  .. code-block:: 
+
+     realm-cli pull --remote="<Your App ID>"
+  
+---
+title: Generate Data Models Based On Your Schema
+ref: pull-the-latest-version-of-your-app
+content: |
+  To begin generating the data object models for your app, run the following: 
+
+  .. code-block:: 
+
+     realm-cli schema datamodels -a "<Your App ID>"
+  
+---
+title: Confirm Options for Generating Data Models
+ref: confirm-options-for-generating-data-models
+content: |
+  You will be prompted for the following details:
+
+  .. code-block::
+
+    ? Select the language you would like to generate data models in
+    ? Would you like to omit imports? 
+    ? Would you like group all generated data models together?
+  
+  1. Select from the list of available languages which language you would like 
+     your object models to be in
+
+  #. Choose whether you'd like to omit import statements from your data models
+
+  #. Choose whether you'd like the data model for each schema to be 
+     grouped together in the output
+  
+---
+title: Use Your Data Model
+ref: use-your-data-model
+content: |
+  Copy and paste the object models from your terminal into your code.
+
+---
```

nit: add a newline to the end of files
```suggestion
---

```